packages/flutter_tools/lib/src/android/application_package.dart

```diff
@@ -286,6 +321,16 @@ class _Attribute extends _Entry {
 class ApkManifestData {
   ApkManifestData._(this._data);
 
+  static String? _getActivityName(_Element? element) {
+    if (element == null) {
+      return null;
+    }
+    final _Attribute? nameAttribute = element.firstAttribute('android:name');
+    // "io.flutter.examples.hello_world.MainActivity" (Raw: "io.flutter.examples.hello_world.MainActivity")
+    return nameAttribute?.value?.substring(1, nameAttribute.value?.indexOf('" '));
```

Comment:
How is "raw" different from the "not raw"? Why do you substring from index 1?

Reply:
It's already in the master branch. I just extracted it as a function. So, I don't know why it is like this.

https://github.com/flutter/flutter/blob/112e8b55095cb75c5061d89ba4cb64cc060f102c/packages/flutter_tools/lib/src/android/application_package.dart#L384-L386